genon/preprocessor/facade/convert_processor.py

```
@@ -2410,6 +1931,26 @@ def __init__(self, config_path: str | None = None):
+            fallback_api_url=ec.api_url,
+            fallback_api_key=ec.api_key,
+            fallback_model=ec.model,
+            config_dir=self._config_dir,
+        )
+        # 런타임 kwargs 오버라이드의 기준(base) 옵션 보관
+        self._base_image_description_options = self.image_description_options
+        # chart.enable=true 이면 그림 분류를 켠다(런타임 chart_detection=auto 전환 허용).
+        if self.image_description_options.chart_enabled:
+            try:
+                self.pipe_line_options.do_picture_classification = True
+            except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)